docs/server/commands.rst

```diff
@@ -462,11 +462,12 @@ no actual data will be loaded:
 
 
 This command is capable of importing translations from several files in
-different formats at once:
+different formats at once, either specified directly or by passing its
+container directory:
 
 .. code-block:: bash
 
-   (env) $ pootle update_tmserver --tm=mozilla --project="Foo 1.7" foo.po bar.tmx eggs.xliff bacon.ts
+   (env) $ pootle update_tmserver --tm=mozilla --project="Foo 1.7" foo.po bar.tmx fr/ eggs.xliff bacon.ts
```

Comment:
Now we have dirs I think this line is confusing, rather limit it to two normal files and one dir at the end